supervisor/docker/network.py

```diff
@@ -42,55 +68,74 @@ def containers(self) -> list[str]:
     @property
     def gateway(self) -> IPv4Address:
         """Return gateway of the network."""
-        return DOCKER_NETWORK_MASK[1]
+        return DOCKER_IPV4_NETWORK_MASK[1]
 
     @property
     def supervisor(self) -> IPv4Address:
         """Return supervisor of the network."""
-        return DOCKER_NETWORK_MASK[2]
+        return DOCKER_IPV4_NETWORK_MASK[2]
 
     @property
     def dns(self) -> IPv4Address:
         """Return dns of the network."""
-        return DOCKER_NETWORK_MASK[3]
+        return DOCKER_IPV4_NETWORK_MASK[3]
 
     @property
     def audio(self) -> IPv4Address:
         """Return audio of the network."""
-        return DOCKER_NETWORK_MASK[4]
+        return DOCKER_IPV4_NETWORK_MASK[4]
 
     @property
     def cli(self) -> IPv4Address:
         """Return cli of the network."""
-        return DOCKER_NETWORK_MASK[5]
+        return DOCKER_IPV4_NETWORK_MASK[5]
 
     @property
     def observer(self) -> IPv4Address:
         """Return observer of the network."""
-        return DOCKER_NETWORK_MASK[6]
+        return DOCKER_IPV4_NETWORK_MASK[6]
+
+    def _get(self, network_id: str) -> docker.models.networks.Network:
+        """Get a network by ID."""
+        return self.docker.networks.get(network_id)
 
-    def _get_network(self) -> docker.models.networks.Network:
+    def _create(self, **kwargs: Any) -> docker.models.networks.Network:
+        """Create a new network."""
+        return self.docker.networks.create(**kwargs)
+
+    def _get_network(self, enable_ipv6: bool = False) -> docker.models.networks.Network:
```

Comment:
This now all happens in the constructor, which is not ideal.

Let's introduce a `async def post_init` and call it from `DockerAPI`'s `post_init`. With that we can also run the API calls in a executor, since those are blocking calls.